fladle-plugin/src/main/java/com/osacky/flank/gradle/FlankGradleExtension.kt

```diff
@@ -29,7 +29,7 @@ open class FlankGradleExtension @Inject constructor(objects: ObjectFactory) : Fl
   override val useOrchestrator: Property<Boolean> = objects.property<Boolean>().convention(false)
   override val autoGoogleLogin: Property<Boolean> = objects.property<Boolean>().convention(false)
   override val devices: ListProperty<Map<String, String>> = objects.listProperty<Map<String, String>>().convention(listOf(mapOf("model" to "NexusLowRes", "version" to "28")))
-
+  override val dependOnAssemble: Property<Boolean> = objects.property<Boolean>().convention(false)
```

Comment:
can this be added at the end of the list beneath `additionalGcloudOptions`? so that it is consistent with the API documentations